src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -56,6 +56,7 @@ def serialize(self, obj, attrs, user):
             "includeAllProjects": obj.include_all_projects,
             "dateModified": obj.date_modified,
             "dateCreated": obj.date_added,
+            "createdBy": obj.created_by,
```

Comment:
We should handle this in `get_attrs` since it'll make a separate query for every `Incident` row.

Reply:
Okay, I can bulk fetch it for this serializer, but I am thinking I will leave the property in so it could be used as a sort field.